packages/api/src/api.gen.ts

```diff
@@ -396,6 +422,7 @@ export interface SendPasswordlessLinkReturn {
   status: boolean
 }
 export interface FriendListArgs {
+  nickname: string
```

Comment:
nickname should not be a required property

Reply:
Passing an empty string will work.

Reply:
Thats what im doing for now, but we should correct this

Reply:
yes, change the RIDL to make it optional @LukasJenicek

Reply:
Totally forgot about this one. It's merged  `stack/api` :https://github.com/0xsequence/stack/pull/774 and deployed: https://buildkite.com/horizon/sequence-stack-build/builds/1068

Reply:
I just did not like the idea that we have to reference string value by pointer, but I think there is nothing I can do about it